activity.js

```diff
@@ -0,0 +1,25 @@
+class Activity {
+  constructor(category, description, minutes, seconds, completed, id) {
+    this.category = category
+    this.description = description
+    this.minutes = minutes
+    this.seconds = seconds
+    this.completed = completed
+    this.id = id
```

Comment:
can you please add ";" to the ends of lines 3-8

Reply:
oops!! sorry!!